mtp_api/apps/core/tests/test_dump_for_ap.py

```diff
@@ -0,0 +1,85 @@
+import csv
+import datetime
+import io
+from unittest import mock
+
+from django.core.management import CommandError, call_command
+from django.test import TestCase, override_settings
+from django.test.utils import captured_stdout
+from django.utils import timezone
+from model_mommy import mommy
+
+from core.tests.utils import make_test_users
+from credit.constants import CREDIT_RESOLUTION
+from credit.models import Credit
+from disbursement.models import Disbursement
+from disbursement.tests.utils import generate_disbursements
+from payment.constants import PAYMENT_STATUS
+from payment.models import Payment
+from payment.tests.utils import generate_payments
+from prison.models import PrisonerLocation
+from prison.tests.utils import load_random_prisoner_locations
+from security.models import PrisonerProfile, SenderProfile, DebitCardSenderDetails
+
+
+class DumpForAPTestCase(TestCase):
+    fixtures = ['initial_types.json', 'test_prisons.json', 'initial_groups.json']
+
+    def basic_setup(self):
+        make_test_users()
+        load_random_prisoner_locations()
+
+    def test_invalid_arguments(self):
+        with self.assertRaises(CommandError):
+            call_command('dump_for_ap')
+        with self.assertRaises(CommandError):
+            call_command('dump_for_ap', 'notifications')
+        with self.assertRaises(CommandError):
+            call_command('dump_for_ap', 'credits')
+        with self.assertRaises(CommandError):
+            call_command('dump_for_ap', 'credits', '-', after='tomorrow')
+        with self.assertRaises(CommandError):
+            call_command('dump_for_ap', 'credits', '-', before='2019-08-01', after='2019-09-01')
```

Comment:
It would be good to say why we expect each of these tests to fail, a comment would do for example.